app/main/routes.py

```diff
@@ -103,19 +104,47 @@ def poc_search():
                 query, current_app.config["AWS_OPEN_SEARCH_INDEX"]
             )
         )
-        results = open_search_response["hits"]["hits"]
-        session["search_results"] = results
-
-    num_records_found = len(results)
+        search_results = open_search_response["hits"]["hits"]
+        session["search_results"] = search_results
 
+    num_records_found = len(search_results)
+    print(num_records_found)
     return render_template(
         "poc-search.html",
         form=form,
-        results=results,
+        results=search_results,
         num_records_found=num_records_found,
     )
 
 
+@bp.route("/poc-transferring-bodies", methods=["POST", "GET"])
+@access_token_login_required
+def poc_transferring_bodies():
+    form = SearchForm()
+    field_name = "Source_Organization.keyword"
+
+    # check if user has transferring body group access or not
```

Comment:
Could we do without the comments in this function? This variable name and function call here seems descriptive enough to not need a descriptive comment which would need maintaining. if anything changes.

and `# get unique transferring bodies` can be replaced with a private function if we want to make that bit of the logic clear:

```
def get_unique_transferring_bodies():
    user_transferring_body_groups = get_user_transferring_body_groups()
    unique_response = search_logic.generate_open_search_client_and_unique_values(
        current_app.config["AWS_OPEN_SEARCH_INDEX"], field_name)
    unique_values = unique_response["aggregations"]["unique_values"]["buckets"]
    unique_transferring_bodies = []
    for value in unique_values:
        group_name = value["key"]
        if len(user_transferring_body_groups) > 0:
            for user_group in user_transferring_body_groups:
                if user_group.strip().replace(' ', '').lower() == group_name.strip().replace(' ', '').lower():
                    unique_transferring_bodies.append(group_name)
    return unique_transferring_bodies
```

Reply:
as discussed over the call, the function logic is temporary and will move once full browse view functionality done